.github/workflows/build_documentation.yml

```diff
@@ -0,0 +1,20 @@
+name: Build Documentation 
+
+on:
+  pull_request:
+    branches: [ master, candidate, gh-pages ]
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+    - name: Repository checkout
+      uses: actions/checkout@v2
+    - name: Build Documentation
+      run: |
+        bundle install
+        echo "" >> MiniApp.podspec # add a new line to the podspec file to remove it after awk treatment
+        awk '!/Admob'\''/' RS=miniapp.subspec ORS=miniapp.subspec MiniApp.podspec > MiniApp.jazzy.podspec # create the jazzy podspec
+        sed -i '' -e '$ d' MiniApp.jazzy.podspec # clean the last line which contains an awk artifact
+        sed -i '' -e '$ d' MiniApp.podspec # remove the last line we added before
```

Comment:
this script might be extracted into a file for a better maintenance

Reply:
Done 👍🏻